test/ftp_object.jl

```diff
@@ -417,44 +418,46 @@ end
 
 @testset "verbose" begin
 
-    function test_captured_ouput(func::Function)
-        local resp
+    function captured_size(func::Function)
         temp_file_path, io = mktemp()
         ftp_init()
         try
-            resp = func(io)
+            func(io)
             close(io)
```

Comment:
```suggestion
```

Not related to this PR, but `close(io)` is already in the `finally`

Reply:
I think the `close` is here to ensure that the IO is flushed so that the filesize check is correct

Reply:
Could you call `flush` directly for that?